packages/messaging/src/helpers/externalizePayload.ts

```diff
@@ -60,6 +60,11 @@ function propagateNotificationPayload(
   if (!!image) {
     payload.notification!.image = image;
   }
+
+  const icon = messagePayloadInternal.notification!.icon;
```

Comment:
I see this is just matching the style from above. In the future, I think using the `?.` instead of `!.` will be better

Reply:
sg. will make a pr to batch fix